.github/workflows/build_packages.yaml

```diff
@@ -493,14 +495,14 @@ jobs:
       with:
         path: |
           source/_build/default/lib/rocksdb/
-          source/deps/rocksdb//
+
         key: ${{ matrix.os }}-${{ matrix.otp }}-${{ matrix.arch }}-${{ needs.prepare.outputs.DEP_ROCKSDB_REF }}
     - name: load quicer cache
       uses: actions/cache@v2
       with:
         path: |
           source/_build/default/lib/quicer/
-          source/deps/quicer/
+
```

Comment:
We'd need to restore the cache as is done [above](https://github.com/emqx/emqx/pull/6909/files#diff-d2796f5d4197787c17ee689a8fc9bed7cb4d73cea4af047c47028e6a5e4f5f9aR399), now that we removed this line, right?

Reply:
yes. We need single source of cache. 
image regular build loaded cache built by elixir that it could not be reused or the other way around.

I am not 100% sure but I will monitor the builds to see if it is true before any complex changes.

Reply:
I revert the change due to https://github.com/emqx/emqx/pull/6909#discussion_r797383198